lib/qesapdeployment.pm

```diff
@@ -1216,20 +1241,17 @@ sub qesap_az_vnet_peering_delete {
     }
 
     record_info('Attempting peering destruction');
-    my $peering_cmd = "az network vnet peering delete -n $peering_name";
     my $source_ret = 0;
     record_info('Destroying job_resources->IBSM peering');
     if ($args{source_group}) {
         my $source_vnet = qesap_az_get_vnet($args{source_group});
-        my $source_cmd = "$peering_cmd --resource-group $args{source_group} --vnet-name $source_vnet";
-        $source_ret = script_run($source_cmd, timeout => $args{timeout});
+        $source_ret = qesap_az_simple_peering_delete(rg => $args{source_group}, vnet_name => $source_vnet, peering_name => $peering_name, timeout => $args{timeout});
     }
     else {
         record_info('NO PEERING', "No peering between job VMs and IBSM - maybe it wasn't created, or the resources have been destroyed.");
```

Comment:
Not for this PR

```
maybe it wasn't created, or the resources have been destroyed.
```

can be misleading, the code falls here just because the called does not provide `source_group`. It could be for any reason, it is not possible for this layer of code to tell why.